Sources/GalleryZoomTransitionController.swift

```diff
@@ -12,15 +12,14 @@ open class GalleryZoomTransitionController: NSObject, UIViewControllerTransition
     open weak var sourceTransition: GalleryZoomTransitionDelegate?
     open var prepareToDismiss: (() -> Void)?
 
+    public init(sourceTransitionDelegate sourceTransition: GalleryZoomTransitionDelegate) {
+        self.sourceTransition = sourceTransition
+    }
+
     open func animationController(
         forPresented presented: UIViewController, presenting: UIViewController, source: UIViewController
     ) -> UIViewControllerAnimatedTransitioning? {
-        guard
-            let sourceTransition = source as? GalleryZoomTransitionDelegate,
```

Comment:
Let's have nil-coalescing here. If sourceTransition is provided, use it, if not, use source view controller